test/cluster_abnormal_state_test.rb

```diff
@@ -30,6 +30,24 @@ def test_the_state_of_cluster_failover
     end
   end
 
+  def test_the_state_of_cluster_node_failure
+    redis_cluster_fail_master do
+      assert_raises(Redis::CannotConnectError, 'Error connecting to Redis on 127.0.0.1:7002') do
+        r.set('key0', 0)
+      end
+
+      100.times do |i|
```

Comment:
do we need `100.times` ? CI is not particularly fast, so I'd rather not do too much unnecessary round trips.

Reply:
updated this and other test 👍